pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -231,6 +231,7 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 	r.status.SetDaemonSets(daemonSets)
 	r.status.SetDeployments(deployments)
 	r.status.SetRelatedObjects(relatedObjects)
+	r.status.SetFromPods()
```

Comment:
Calling `SetFromPods()` before reconciling the objects will mean that you'll get a `Progressing` status with a bunch of warnings like "`Error getting DaemonSet openshift-sdn/sdn: Not Found`". That will get cleared eventually, but we prefer to not temporarily set incorrect statuses.

Moving this to *after* the `Apply the object to the cluster` loop should fix that.

Reply:
Thanks for the comment, just updated my PR